data/governments.txt

```diff
@@ -131,6 +131,62 @@ government "Free Worlds"
 	"hostile hail" "hostile free worlds"
 	raid "pirate raid"
 
+government Gegno
+	swizzle 28
+	color 0.56 0.44 0.2
+	"player reputation" 0.01
+	"crew attack" 1.7
```

Comment:
The current highest crew attack is for the Alphas, 1.5. Not sure how big of an issue this is, but just something to keep in mind. Same goes for the other Gegno governments.

Reply:
Noted- will knock that down. Defense would be more their strongpoint anyway.